src/Xrefcheck/CLI.hs

```diff
@@ -142,6 +158,13 @@ traversalOptionsParser = do
     help "Files and folders which we pretend do not exist."
   return TraversalOptions{..}
 
+verifyOptionsParser :: Parser VerifyOptions
+verifyOptionsParser = do
+  voIgnoreLocalhost <- flag True False $
+    long "check-localhost" <>
+    help "If provided localhost links will be checked"
```

Comment:
Minor nitpick: we should be consistent about trailing dots :stuck_out_tongue_closed_eyes: 

Same in some other places.